spec/features/visitors/password_recovery_spec.rb

```diff
@@ -335,4 +358,16 @@ def reset_password_and_sign_back_in(user)
     expect(page).to have_content t('notices.password_reset')
     expect(ActionMailer::Base.deliveries).to be_empty
   end
+
+  # @return [Array(User,String)]
+  def loa3_user_with_recovery_code(pii)
+    user = create(:user, :signed_up)
+    profile = create(:profile, :active, :verified, user: user)
+    pii_attrs = Pii::Attributes.new_from_hash(pii)
+    user_access_key = user.unlock_user_access_key(user.password)
+    recovery_code = profile.encrypt_pii(user_access_key, pii_attrs)
+    profile.save!
+
+    [user, recovery_code]
```

Comment:
I'll split up making the user, and forward-encrypting the PII which gives us the recovery code